### PR TITLE
Extract token

### LINKED ISSUE #9: Extract authtoken from url using content script
An authorisation will redirect user to our hosted GitHub page with auth token in URL, we will run content script on that page (see manifest) and retrieve auth token from URL.

### PR DESCRIPTION
**Pull request by thisiseshan**
Fixes #9

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   },
   "content_scripts": [
     {
-     "matches": ["https://wdlsvnit.github.io/paper-extension/*"],
+     "matches": ["<all_urls>"],
```
Comment (AdityaHirapara) [inline manifest.json]: Don't change this.

File: contentScript.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
   if (token) {
     chrome.storage.sync.set({ 'token': token });
   }
 })();
 
 function extractTokenFromURL(url) {
-  //TODO: get authtoken from url
-}
\ No newline at end of file
+  var m = url.match(/[#?](.*)/);
```
Comment (AdityaHirapara) [inline contentScript.js]: Rename variable to something meaningful.

File: contentScript.js [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
+  if (!m || m.length < 1)
+    console.log("Get a url");
```
Comment (AdityaHirapara) [inline contentScript.js]: Return empty string instead
And also put braces for if block.

**Comment by lunaticmonk** [comment]
@thisiseshan good work.

**Pull request merged**